### PR TITLE
Portfolio: Popup window

### PR DESCRIPTION
**Pull request by Masuduzzaman811**
## Details Popup window
In this new branch, I've implemented popup windows to see project details. 

### Key Requirements:
- A popup window appears by clicking or tapping the "See Project" button
- The popup window disappears by clicking or tapping the close (X) button.
- Should implement both mobile and desktop popup window.
- Use an array to store all information for the projects.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -160,173 +133,394 @@ <h2 class="card-head">Multi-Post <br>Stories</h2>
...
+                <p class="card-paragraph" id="p1-p"></p>
+    
+                <ul class="card-lang-list modal-lang-list">
+                    <li class="card-lang-list-item" id="p1-lan1"></li>
+                    <li class="card-lang-list-item" id="p1-lan2"></li>
+                    <li class="card-lang-list-item" id="p1-lan3"></li>
+                </ul>
+                <div class="ex-margin">
+                    <a class="card-button-link button"  href="#" rel="noopener noreferrer" target="_blank">See Live <img src="images/live-btn.svg" alt="icon"></a>
+                    <a class="card-button-link button"  href="#" rel="noopener noreferrer" target="_blank">See Source <img src="images/git-btn.svg" alt="icon"></a>
```
Comment (ShoiraTa) [inline index.html]: - [x] Please check these buttons on popup modal. They are wrapping in screen sizes smaller than `1075px`, including mobile size and look strange 😁 

Desktop  | Mobile
------------- | -------------
Screenshot 2022-04-15 at 16 00 16 | Screenshot 2022-04-15 at 15 59 56

- [x] In desktop size, please add adividing line between the buttons and tech tags

Your design  | Figma design
------------- | -------------
Screenshot 2022-04-15 at 16 01 45  |  Screenshot 2022-04-15 at 16 01 20

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi team,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

- [x] This is a collaborative project, therefore please make equal contributions. I can see that @wale-prog  has only 1 contribution in this project while @medsonmoombe has 7. This looks very unequal 😁 

- [x] Check more the comments under the review.

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Masuduzzaman811) [reply index.html]: - Modal buttons are now responsive
- A dividing line has been added after the technology tag list

**Review by vic778 — APPROVED** [review]
Hi @Masuduzzaman811   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**